flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/table/stream/StreamingFileWriterTest.java

```diff
@@ -60,25 +58,26 @@
 import static org.apache.flink.connector.file.table.FileSystemConnectorOptions.SINK_PARTITION_COMMIT_POLICY_KIND;
 import static org.apache.flink.connector.file.table.FileSystemConnectorOptions.SINK_PARTITION_COMMIT_TRIGGER;
 import static org.apache.flink.connector.file.table.FileSystemConnectorOptions.SINK_PARTITION_COMMIT_WATERMARK_TIME_ZONE;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /** Test for {@link StreamingFileWriter}. */
-public class StreamingFileWriterTest {
+class StreamingFileWriterTest {
 
-    @ClassRule public static final TemporaryFolder TEMPORARY_FOLDER = new TemporaryFolder();
     private final OutputFileConfig outputFileConfig = OutputFileConfig.builder().build();
     private final DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
 
+    @TempDir private java.nio.file.Path tmpDir;
     private Path path;
 
-    @Before
-    public void before() throws IOException {
-        File file = TEMPORARY_FOLDER.newFile();
+    @BeforeEach
+    void before() throws IOException {
+        File file = tmpDir.toFile();
         file.delete();
         path = new Path(file.toURI());
```

Comment:
```suggestion
        path = new Path(tmpDir.resolve("tmp").toUri());
```
his part is a bit strange.